src/applications/resources-and-support/components/SearchBar.jsx

```diff
@@ -55,8 +62,7 @@ function SearchBar({ onInputChange, previousValue, setSearchData, userInput }) {
       : `${resourcesSettings.rootUrl}/`;
     const newUrl = `${URL}?${queryParams}`;
 
-    history.replaceState({}, '', newUrl);
-    window.location.href = newUrl;
+    window.location.assign(newUrl);
```

Comment:
The current way this search bar was going to the `/search` page, it was rewriting the history. When you'd land on `/search` after using the `/resources` search bar, the browser back button was disabled. There wasn't an apparent reason for this, so I adjusted it so the back button will work.